headers/types/dungeon_mode/enums.h

```diff
@@ -1410,4 +1410,24 @@ enum number_color {
     NUMBER_COLOR_LIGHT_BLUE = 11,
 };
 
+// Used in top_screen_status to store what kind of fade the screen is currently in, if any.
+enum top_screen_fade_status {
+    TOP_SCREEN_BLACK = 2,
+    TOP_SCREEN_FADE_IN = 4,
+    TOP_SCREEN_NO_FADE = 5,
+    TOP_SCREEN_FADE_OUT = 6
```

Comment:
Unknown/unused values should be documented as well. Also, I think the prefix could be updated to `TOP_SCREEN_FADE`, for clarity.

Something like:

```suggestion
    TOP_SCREEN_FADE_0x0 = 0,
    TOP_SCREEN_FADE_0x1 = 1,
    TOP_SCREEN_FADE_BLACK = 2,
    TOP_SCREEN_FADE_0x3 = 3,
    TOP_SCREEN_FADE_FADE_IN = 4,
    TOP_SCREEN_FADE_NO_FADE = 5,
    TOP_SCREEN_FADE_FADE_OUT = 6
```